modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/java/com/liferay/frontend/taglib/clay/sample/web/internal/display/context/ClaySampleDisplayContext.java

```diff
@@ -111,6 +111,11 @@ public List<TabsItem> getTabsItems() {
 				tabsItem.setLabel("Vertical Nav");
 				tabsItem.setPanelId("vertical_nav");
 			}
+		).add(
+			tabsItem -> {
+				tabsItem.setLabel("Search Paginator");
+				tabsItem.setPanelId("search_paginator");
+			}
```

Comment:
Similarly here, this code does not belong to this sample

Reply:
Done!